### ISSUE TITLE
Add wall grab/jump sprites and hurtboxes

### ISSUE THREAD

**Issue by flip40** [DemoProductions/fighter#89]
Add proper sprites and hurtboxes for wall jumping.

**Comment by ghost** [comment]
The two wallgrab/jump sprites are on the 2nd to the bottom and to the right, right?

https://www.facebook.com/download/1535985176725410/SpriteSheet.Final.12.5.png

**Comment by flip40** [comment]
Yes, though it should lead into one or two of the regular jump sprites (wj1 wj2, j3, j4, j5 or something like that).. Also we need to mirror the first wall jump sprite as it is also the wall grab sprite.. it faces opposite of the last moved direction. Gimme a minute, I'll edit it.

**Comment by flip40** [comment]
Use this.
![spritesheet final 12 5](https://cloud.githubusercontent.com/assets/10754057/11706034/0958ee84-9eaa-11e5-99a2-c165aab599d7.png)

**Comment by ghost** [comment]
Ok, I won't be working on this until tomorrow morning after an early final (so around 10). So the first jump sprite grabs on to the the wall, and then the 2nd one kicks off right?

**Comment by flip40** [comment]
Yes.
